app/views/hotspots/edit.html.erb

```diff
@@ -1 +1,3 @@
-<%= render 'form' %>
\ No newline at end of file
+<%= render 'form' %>
+<%= link_to 'Delete'%>
+    flash[:success] = "The hotspot was deleted succesfully"
```

Comment:
the only reason I put this here was when I was running the test it kept saying it needed it. Once I put that there it gave me a green